custom_components/ha_creality_ws/sensor.py

```
@@ -893,18 +894,34 @@ def add_cfs_entities():
+    # `call_soon` cannot be cancelled, and disconnecting the dispatcher does not
+    # unschedule a callback that is already queued. Without this flag the
+    # deferred `async_add_entities` could run against an unloaded entry.
+    platform_live = True
+
+    def _mark_unloaded() -> None:
+        nonlocal platform_live
+        platform_live = False
+
+    entry.async_on_unload(_mark_unloaded)
+
+    def _add_if_live(new_ents):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_add_if_live`

Add return type annotation: `None`

(ANN202)